src/metabase/automagic_dashboards/core.clj

```diff
@@ -551,7 +664,7 @@
                        :dashboard_filters
                        (mapcat (comp :matches (:dimensions context))))
         cards     (make-cards context rule)]
-    (when cards
+    (when (or cards (-> rule :cards nil?))
```

Comment:
From reading `make-cards`, I'm thinking that above should be `(seq cards)`. It looks like `make-cards always returns a seq, but will possibly be empty.

Reply:
(apply merge-with identity []) returns nil which then short circuits due to some->>, but writing this made it clear, that a (not-empty cards) will be much more descriptive ;)